### ISSUE TITLE
given iso offset is coerced by dst-offset

### ISSUE THREAD

**Issue by davyuu** [spencermountain/spacetime#103]
![image 1](https://user-images.githubusercontent.com/3384239/53274597-cb5f7a00-36c5-11e9-8d51-d32de4f7bfa6.png)

Hey, bumped into another issue with parsing ISO timestamps. In this case, I am trying to parse `2019-03-13T18:00:00.000-05:00` which is in CST or GMT-6 during DST which is in `UTC-5`. From the screenshot, I am just parsing the ISO timestamp and outputting it in ISO format and it is converting the offset to `-4:00` while keeping the time the same.

I would really appreciate it if this and [Issue 101](https://github.com/spencermountain/spacetime/issues/101) could be pushed into a release build. Thanks!

**Comment by hunteva** [comment]
![image](https://user-images.githubusercontent.com/491380/53285106-b6681100-37c1-11e9-92b1-b9f43dd6ba3d.png)

Could not reproduce this with version 5.2.1, can you please confirm what was your local tz and the version?
Cheers

**Comment by davyuu** [comment]
![image](https://user-images.githubusercontent.com/3384239/53288705-9ef53d80-3759-11e9-9bc0-0337a4f308e4.png)

I was able to recreate this on my other machine using the latest version `5.2.2`.
My local tz is Toronto EST. It might be related to the fact that the parsed tz and my local tz are both in DST during that timestamp.

**Comment by hunteva** [comment]
Found the cause, fixed in #105

**Comment by davyuu** [comment]
@hunteva @spencermountain Thanks for fixing the issue! Sorry to bother, but is it possible to push a version with this fix?

**Comment by spencermountain** [comment]
i think this made it into 5.3.0

**Comment by spencermountain** [comment]
oops, no it didn't.
i've got something dirty on dev right now, will take a look at it in the next day or two

**Comment by davyuu** [comment]
@spencermountain Thanks!

**Comment by spencermountain** [comment]
published `5.4.0`, 
thanks for your help!

**Comment by spencermountain** [comment]
hey, i've re-introduced this issue, how fun!
at least, it will trigger if you set your system clock before dst.
will take a look at it this week
`:/`

**Comment by spencermountain** [comment]
```
// bug 1: set clock before march
let a = spacetime('2019-03-13T18:00:00.000-05:00')
console.log(a.format('iso'))
// 2019-03-13T18:00:00.000-04:00
```

**Comment by spencermountain** [comment]
closed again, in `5.7.0`
thanks
